stdlib/number.gr

```diff
@@ -86,6 +86,24 @@ export let rec sqrt = (x: Number) => {
   ret
 }
 
+/**
+ * Determine the positivity or negativity of a Number.
+ *
+ * @param x: The number to inspect
+ * @returns `-1` if the number is negative, `1` if positive, or +/-`0` otherwise
```

Comment:
Should we revise this?

Reply:
Is it worth the verbosity? How would you want to state it?

Reply:
Probably not. Two options I'll let you choose from:

```suggestion
 * @returns `-1` if the number is negative, `1` if positive, or `0` otherwise
```

```suggestion
 * @returns `-1` if the number is negative, `1` if positive, or `0` otherwise. Signedness of `-0.0` is preserved.
```

Reply:
I actually like that 2nd one but we should use a semicolon and no periods for the returns
```suggestion
 * @returns `-1` if the number is negative, `1` if positive, or `0` otherwise; signedness of `-0.0` is preserved
```
Thoughts?

Reply:
Works for me!